apps/src/applab/applab.js

```diff
@@ -1332,10 +1335,17 @@ Applab.onCodeModeButton = function() {
 /**
  * If the filename is relative (contains no slashes), then prepend
  * the path to the assets directory for this project to the filename.
+ *
+ * If the filename is external and non-https, route it through the MEDIA_PROXY.
  * @param {string} filename
  * @returns {string}
  */
 Applab.maybeAddAssetPathPrefix = function (filename) {
+
+  if (/^http:\/\//.test(filename)) {
+    return MEDIA_PROXY + encodeURIComponent(filename);
+  }
```

Comment:
I see that there is some test coverage below.  Is this adequate to cover all of the cases we care about?  (http: vs https, external vs. internal.)